File: train.rb
require 'pycall/import'
include PyCall::Import

pyimport 'gensim'

sentences = gensim::models::word2vec.LineSentence.new('text/wakachied.txt') 

model = gensim::models::word2vec.Word2Vec.new(
  sentences,
  sg: 1,
  size: 100,
  min_count: 1,
  window: 10,
  hs: 1,
  negative: 0
)

model.save('train.model')

File: train.py
from gensim.models import word2vec
import logging
import sys

def train(text_file_path, model_file_path):
    logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=logging.INFO)

    sentences = word2vec.LineSentence(text_file_path)
    model = word2vec.Word2Vec(sentences,
                          sg=1,
                          size=100,
                          min_count=1,
                          window=10,
                          hs=1,
                          negative=0)
    model.save(model_file_path)
